config/locales/nl.yml

```diff
@@ -456,37 +456,37 @@ nl:
         voice_chat: 'spraak oproepen'
         training: training
         directory: lijst
-        self_injury: 'zelf verminking'
+        self_injury: 'zelfverminking'
         treatment: behandeling
-        care_planning: 'zorg planning'
+        care_planning: 'zorgplanning'
         depression: depressie
-        ocd: OCD
+        ocd: OCS
         advice: advies
         forum: forum
         learning: leren
         crisis_prevention: 'crisis preventie'
-        safety_planning: 'veiligheids planning'
-        suicide_prevention: 'suicide preventie'
-        android: android
+        safety_planning: 'veiligheidsplanning'
+        suicide_prevention: 'zelfmoordpreventie'
+        android: Android
         ios: iOS
-        support_groups: 'ondersteuning groepen'
+        support_groups: 'ondersteuninggroepen'
         mindfulness: mindfulness
         journaling: dagboeken
         mood_tracking: 'stemmings tracking'
         veterans: veteranen
         music: muziek
         integrative: integratief
-        ptsd: PTSD
+        ptsd: PTSS
```

Comment:
PTSS are usually symptoms and PTSD are usually referred as a full disorder in Psychology. I prefer to keep PTSD. The same distinction is made everywhere as far as I know

Reply:
In school and in my internships in Belgium we always referred to PTSS since that is just the Dutch version of PTSD. I've never seen the practice that PTSD is for the full disorder and PTSS for the symptoms. That seems quite odd in my opinion. But I guess most people know the two terms so if you like to keep PTSD, that's fine with me 👍